src/org/loklak/api/search/SearchServlet.java

```diff
@@ -106,18 +109,18 @@ public static Timeline search(final String protocolhostportstub, final String qu
         //System.out.println(parser.text());
         return tl;
     }
-    
+
     @Override
     protected void doPost(final HttpServletRequest request, final HttpServletResponse response) throws ServletException, IOException {
         doGet(request, response);
     }
-    
+
     @Override
     protected void doGet(final HttpServletRequest request, final HttpServletResponse response) throws ServletException, IOException {
         final long start = System.currentTimeMillis();
         final Query post = RemoteAccess.evaluate(request);
         final ClientIdentity identity = AbstractAPIHandler.getIdentity(request, response, post);
-        
```

Comment:
Please revert this line, it's not part of the issue. If you want to do some house-keeping or cleaning, please create a new PR and Issue for it.